org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/util/IResourcesSetupUtil.java

```diff
@@ -246,6 +248,15 @@ protected void execute(IProgressMonitor monitor)
 		return file;
 	}
 
+	public static File createTempFile(String fileName, String suffix, String content)
+			throws Exception {
+		File file = File.createTempFile(fileName, suffix);
+		FileWriter writer = new FileWriter(file);
```

Comment:
It's always surpising if a closable is not closed in a finally block

Reply:
Fixed.